src/data/taskList.ts

```diff
@@ -66,3 +66,20 @@ export const bizCaseStatus = (
       return 'CANNOT_START';
   }
 };
+
+export const chooseBusinessCaseLink = (
+  bizCase: BusinessCaseModel,
+  status: string
```

Comment:
Could you be more explicit about the status we're using here? Is it the business case status or the system intake status? Also, if you're only using the business case for the id, then just passing in the ID to this function might make it cleaner and easier to test.

Also, two params is kind of my soft limit for using props that aren't named. I think you're good here, but just adding some color in case you're ever writing functions with 3+ props. It's easier to see which prop is which when calling the function.